docs/api/web-frame-main.md

```diff
@@ -89,6 +89,20 @@ this limitation.
 
 Returns `boolean` - Whether the reload was initiated successfully. Only results in `false` when the frame has no history.
 
+#### `frame.send(channel, ...args)`
```

Comment:
We should have `frame.postMessage` as well, so MessagePorts can be passed.

Reply:
should I do it here or in a separate PR? I would merge this as it's ready a do a follow-up